superstake-ui/src/hooks/useEstimateApr.ts

```diff
@@ -110,11 +110,12 @@ const useEstimateApr = ({
 
 	if (isNaN(solBorrowAmount)) return DEFAULT_VALUE;
 
-	const lstAprFromApy =
-		aprFromApy(lstMetrics.lstPriceApy30d, Math.floor(365 / 2)) / 100;
+	const lstApr = lstMetrics?.lstPriceApr30d
+		? lstMetrics?.lstPriceApr30d / 100
+		: aprFromApy(lstMetrics.lstPriceApy30d, Math.floor(365 / 2)) / 100;
```

Comment:
Not really related to your changes but I often ask myself why we show this in APR not APY? The LSTs auto-compounds and so does the deposit yield, so the APY number is more truthful to what you're actually getting, right? The only thing that would not really be factored in is the BLZE rewards for bSOL because those are not compounded automatically.

Not blocking I just wonder what everyone's thoughts are on it. I feel like the only reason we APR in the first place is because the designs said APR not APY, but I keep wondering if we should change that which would also simplify the code.